README.md

```diff
@@ -5,76 +5,143 @@
 
 HawkTracer is a highly portable, low-overhead, configurable profiling tool built in Amazon Video for getting performance metrics from low-end devices.
 
+HawkTracer is available on most popular platforms: **Linux**, **Windows**, **OS X**. The library can be used with **C** and **C++**.
+
+The library provides many different types of events (e.g. CPU usage event, duration tracepoint), but the list can easily be extended by the user.
+
+## Features
+* Low CPU/memory overhead
+* Multi-platform
+* Highly configurable on runtime
```

Comment:
Any examples of this documented anywhere? I'm not sure what is configurable

Reply:
Unfortunately, it's not documented yet. You can configure size of internal buffers, allocators, timelines (which features should be supported). However I might change it to 
`Highly configurable and extensible`
as you can extend this project in a few ways (implement your own features, streams, listeners).
What do you think?